tap.go

```diff
@@ -53,6 +56,7 @@ func (bucket *WalrusBucket) StartDCPFeed(args sgbucket.FeedArguments, callback s
 	}
 
 	go func() {
+		close(args.DoneChan)
```

Comment:
Same here, we're marking this as done before it's finished.

Reply:
Yeah, that is correct. Changes are included in the latest commit.